_includes/macros/activity-snippets.html

```diff
@@ -42,7 +42,7 @@
 
 {# ==========================================================================
 
-   _activity_snippet()
```

Comment:
:+1: SG, fixed in https://github.com/cfpb/cfgov-refresh/commit/4cf0a2537d336fde83866b5074a7ff716acfcfeb